src/services/dashboards/all-dashboards/AllDashboardsPage.vue

```diff
@@ -170,6 +172,39 @@ export default {
             });
         };
 
+        const combinedDashbaordLabelsAutoCompleteHandler = (): ValueHandler | undefined => {
+            const projectLabelsValueHandler = makeDistinctValueHandler('dashboard.ProjectDashboard', 'labels');
+            const domainLabelsValueHandler = makeDistinctValueHandler('dashboard.DomainDashboard', 'labels');
```

Comment:
Suggestion: This handler seems to require filters for `Viewer` or `Scope` options.